### ISSUE TITLE
WebExtension to evaluate image quality

### ISSUE THREAD

**Issue by gaul** [gaul/gaul#9]
https://sliday.com/works#/retina-chrome-extension/ can detect sub-optimal hi-res images but I would like an extension which also detects wrong/stretched dimensions, too large file sizes, missing ALT text, etc.  Ideally this could limit to only a few domains for testing local sites.  The extension would highlight bad images in transparent red like Chrome Dev Tools highlights ads.
